### PR TITLE
Pubby Whiteship Redo Attempt 2

### PR DESCRIPTION
**Pull request by Addust**
Now with SDMM and slightly less shitty code!

## About The Pull Request

A complete redo of the Pubby Station whiteship to be an actual ship, with some new loot added, such as a mining MOD.
Ship is the same as the last attempt at this PR but with cargo bay
## Why It's Good For The Game

Redoes the pubby station whiteship in its entirety, as it was a shitty ship before. This version is simply put, an upgrade, with all the same loot and some extra things.

## Changelog

This can easily be concretely observed, so I'm going to put a changelog in.
image

:cl:
expansion: Redoes the pubby whiteship in its entirety.
add: Pubby whiteship now has more loot. Have fun with that.
/:cl:

### CODE REVIEW

**Comment by Addust** [comment]
Found the double cable problem gonna cut it out one mo

**Comment by Addust** [comment]
alright, i will begin the redo

**Comment by Addust** [comment]
actually fuck it nuclear solution BEGONE MEDBEAM

**Comment by Addust** [comment]
what the fuck is going on with the map diff renders I DON'T UNDERSTAND

**Comment by Addust** [comment]
> put the module in the suit storage as a variable or in the locker, the way youre doing it will just put the module on the suit storage tile i dont think you need overspace tiles on the corners
> 
> personal opinions: medbeam guns are cancer in any normal round wielded by a non-antagonist the medbay area feels a bit lifeless in its decals and the layout being mostly copies of the same stuff, kits and sleepers i think you would be able to get a better effect from "speccing" more into some department rather than a general sorta ftl looking ship (be it medbay like box's ship, a mining vessel, an engineering repair ship or a security fighter)
> 
> the design feels nice though good job

medbay has been replaced by cargo bay which is packed with surplus shit and a refinery to make the ship's purpose obvious

**Comment by Addust** [comment]
can someone review this

**Comment by Addust** [comment]
s h i t

**Comment by Addust** [comment]
fikou i deleted every reference of overspace

File: _maps/shuttles/whiteship_pubby.dmm [hunk 1]
```diff
@@ -40,7 +40,7 @@
 /turf/open/floor/iron/dark,
 /area/shuttle/abandoned/crew)
 "ah" = (
-/turf/closed/wall/mineral/titanium/overspace,
+/turf/closed/wall/mineral/titanium/,
```
Comment (Fikou) [inline _maps/shuttles/whiteship_pubby.dmm:43]: bad path

Comment (Fikou) [reply _maps/shuttles/whiteship_pubby.dmm:43]: also these keys are likely to not exist since titanium wall and an area is likely to be already defined

Comment (Addust) [reply _maps/shuttles/whiteship_pubby.dmm:43]: ah shit my bad

**Comment by Addust** [comment]
this PR is pain to deal with

**Comment by Addust** [comment]
everything's finished, waiting on review and merge

**Comment by Addust** [comment]
has this PR been forgotten already or are headcoders just waiting for something

**Comment by Addust** [comment]
gonna ping the reviewers since the PR is finalised. @EOBGames @ShizCalev @Maurukas

**Comment by EOBGames** [comment]
Alright, sorry for being so slow on the uptake with this, but it's been a busy few weeks for me. To be frank, we're not interested in the map. While I appreciate that some people don't like the Pubby whiteship for being so small, I think it adds a welcome uniqueness to the whiteship pool.

Now, I do want to address some of the technical issues with the map, because I think it's important that you know what's up with it for any future attempts at a redo with a different overall design:

- The design is just really generic. Generally that's something the whiteships manage to avoid, so I don't particularly want us to regress on that front. I think the concept of a mining ship is fine, but it would help if it actually felt like a mining ship, you know? More of an overall industrial aesthetic. It's also very cramped and undecorated- which doesn't make it a huge upgrade over the original in design or aesthetic.
- I'm really not sure what's going on with the cables in this design. They're all over the shop. You've got them running under walls to seemingly nowhere, they're placed in giant blocks around the place, there's some that are entirely disconnected.
- The lack of atmos isn't really a dealbreaker, but when you're getting to a bigger design like this a small atmos system is a good thing to include.
- The ship is unable to dock with Metastation without taking out at least some of arrivals windows and an external airlock, and on Delta this docking will vent arrivals: 
image
image

I think that's enough to drop in as a basic overview of why we're not interested. If you want to take another stab at a whiteship, we (meaning the mappers and maptainers) are happy to offer guidance on the discord.

**Comment by Addust** [comment]
I'll take a 2nd stab at it. Later. Maybe. Mayhaps. Possibly.

**Pull request closed without merging**